index.Rmd

```diff
@@ -105,4 +105,4 @@ ggPredict(fit,se=TRUE,interactive=TRUE)
 I learned how to prepare data, create plots for final porject.
```

Comment:
of course you'll need to update this to be a more formal conclusion about the data.